app/services/import_responsible_body_users_from_computacenter_csv_service.rb

```diff
@@ -0,0 +1,65 @@
+require 'open-uri'
+require 'csv'
+
+class ImportResponsibleBodyUsersFromComputacenterCsvService
```

Comment:
Not a fan of this class name or that it's a service. Could this be a `ComputacenterCsvFile` model with a `import_responsible_bodies` method?

Reply:
It was for consistency with the rest of the `ImportXYZService` classes - there's a Tech Debt ticket in the backlog to review our usage of the `Service` concept, it's got very muddled.